### PR TITLE
validation/util/container: Use --bundle (and stop requiring BundleDir)

### PR DESCRIPTION
**Pull request by wking**
Address a previous TODO.  And now that we are using `--bundle`, we no longer need to set `cmd.Dir`.  The TODO mentions a lack of runc support, but runc supports `--bundle` since opencontainers/runc@3fe7d7f3 (opencontainers/runc#827).

### CODE REVIEW

**Comment by liangchenye** [comment]
'signed-off-by' required @wking

**Comment by wking** [comment]
On Wed, Jan 10, 2018 at 11:59:55PM -0800, 梁辰晔 (Liang Chenye) wrote:
> 'signed-off-by' required @wking 

Oops.  Fixed with cf5cd7f → 196855e.

**Comment by q384566678** [comment]
@wking  I think you should resubmit the commit.

**Comment by wking** [comment]
Closing and reopening to kick Travis.

File: validation/util/container.go [hunk 1]
```diff
@@ -30,19 +30,26 @@ func NewRuntime(runtimeCommand string, bundleDir string) (Runtime, error) {
 	var err error
 	r.RuntimeCommand, err = exec.LookPath(runtimeCommand)
 	if err != nil {
-		return Runtime{}, err
+		return Runtime{}, clierror.NewError(clierror.Executable, err, "1.0.0")
```
Comment (wking) [inline validation/util/container.go]: Oops,I didn't mean to include `clierror` stuff in this PR.

**Comment by liangchenye** [comment]
LGTM

**Comment by q384566678** [comment]
LGTM

**Pull request merged**